phcovid/phcovid.py

```diff
@@ -78,6 +113,7 @@ def get_cases(
     for col in DATE_COLS:
         df_aliased[col] = df_aliased[col].apply(lambda x: fix_dates(x))
 
+    attach_supplement_data(df_aliased)
```

Comment:
I prefer to not edit the dataframe inplace.

Can we make this something like:
`df_aliased[targets] = attach_supplement_data(df_aliased, targets=targets)`

Reply:
Thanks on this! Gave me an idea how to properly manage the DF. The function `attach_supplement_data` is renamed to `extract_supplement_data` to make it more suitable on what it does.